### ISSUE TITLE
Cannot compile when Bitcode is enabled

### ISSUE THREAD

**Issue by damienrambout** [0xced/XCDYouTubeKit#173]
When Bitcode is enabled (by default on Cocoapods), the following compiling error occurs:
```
ld: -U and -bitcode_bundle (Xcode setting ENABLE_BITCODE=YES) cannot be used together
```
I saw you use `-U` arguments in the `xcconfig` value of your podspec. 

I would really like to enable bitcode for my app, so is there a way to replace these `-U` arguments with something else?

Cheers.

**Comment by 0xced** [comment]
This is a duplicate of #165. I’m going to release version 2.2.1 soon which will address this issue.

**Comment by damienrambout** [comment]
@0xced The fix suggested in #165 only works if you **turn off** Bitcode on the app target / project. 
But it you let it turned on (which is what I want, what is recommended and will probably be required someday), I get the same error:
```
ld: -U and -bitcode_bundle (Xcode setting ENABLE_BITCODE=YES) cannot be used together
```

I'll wait for the 2.2.1 version anyway. 
Cheers.

**Comment by 0xced** [comment]
Just delete all the `-U` linker flags and link the JavaScriptCore framework instead. The `-U` linker flags must be used if you are targeting iOS < 7 as explained in the [README](https://github.com/0xced/XCDYouTubeKit/#installation).

**Comment by damienrambout** [comment]
Can I discard these linker flags when using Cocoapods? Since they are imported from the Podspec, they will automatically be appended to the generated `xcconfig` file.

**Comment by damienrambout** [comment]
I tried installing it using Carthage, but I get a compilation error (when using Xcode 7 compiler) because your project treats Warning as Errors. There are several warnings about unchecked nullability.

**Comment by 0xced** [comment]
Yes, you can manually discard these linker flags if your app is targeting iOS 7 and later.

**Comment by 0xced** [comment]
I just released version 2.3.0, bitcode will work fine.

**Comment by damienrambout** [comment]
@0xced Cool! It seems to work now :) Thanks!
